src/pybind/mgr/dashboard/servers.html

```diff
@@ -24,9 +24,16 @@
             };
 
             rivets.formatters.short_version = function(version) {
-                // Turn "ceph version 1.2.3-g9asdasd (as98d7a0s8d7)"
-                // into "1.2.3-g9asdasd"
-                return /ceph version (.+) \(.+\)$/.exec(version)[1];
+                // Expect "ceph version 1.2.3-g9asdasd (as98d7a0s8d7)"
+                result = /ceph version (.+) \(.+\)/.exec(version);
```

Comment:
The places we pass it around (e.g. daemon metadata) we only have a string.

We could improve the format of the string though -- prefixing with "ceph version" seems redundant.  Once that's removed we could guide people to take everything up to the first space as the short version and treat everything else as detail, perhaps.

@dillaman regex updated